cmd/upgrade/app/v1alpha1/executor.go

```diff
@@ -35,9 +34,6 @@ const (
 	// envSelfNamespace represent namespace of job in which upgrade process is running.
 	// This is required to build owner reference of upgrade result cr.
 	envSelfNamespace = "OPENEBS_IO_SELF_NAMESPACE"
```

Comment:
lets rename SELF to POD in env variable name as well as code variable name

Reply:
done